### ISSUE TITLE
commonauth

### ISSUE THREAD

**Issue by jh-dwolla** [planettelex/splunk-crowd-auth#2]
greetings, I get an error "no module named commonauth".  I also cannot find it with a pip search.  There are a few found with a google search, but which one is correct to use?

Thanks

**Comment by dansomething** [comment]
Take a look in {splunk dir}/share/splunk/authScriptSamples/commonAuth.py.  readInputs() is being used from there.

**Comment by pdjhh** [comment]
Hi there. Sorry I don't know anything about python but need to get this connector working over to crowd. I get the same message but don't understand your answer. What is it I need to do exactly please? I can see that file in that directory. Thanks.

**Comment by pdjhh** [comment]
I'm revisiting this. Does this provide a simple Yes/No return for authentication or can it tell Splunk which AD group the user is a member of as well as our Crowd hooks back into AD. Thanks.
